ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -774,6 +774,22 @@ extension SignalProducerType where Value: Equatable {
 	}
 }
 
+extension SignalProducer where Value: Hashable {
+	/// Forwards only those values from `self` that are unique across the set of
+	/// all values that have been seen
```

Comment:
I would explicitly call out that this retains all values that are sent on `self`. It's obvious once you think about it, but the consequences can be large, so it deserves to be documented.